core-services/importExport/src/main/java/org/silverpeas/core/importexport/model/PublicationType.java

```diff
@@ -23,109 +23,100 @@
  */
 package org.silverpeas.core.importexport.model;
 
-import org.silverpeas.core.importexport.attachment.AttachmentsType;
-import org.silverpeas.core.importexport.versioning.DocumentsType;
-
+import org.silverpeas.core.contribution.publication.model.PublicationDetail;
+import org.silverpeas.core.importexport.attachment.AttachmentDetail;
 import org.silverpeas.core.importexport.coordinates.CoordinatesPositionsType;
-import org.silverpeas.core.node.importexport.NodePositionsType;
-import org.silverpeas.core.pdc.pdc.importexport.PdcPositionsType;
 import org.silverpeas.core.importexport.publication.PublicationContentType;
-
-import org.silverpeas.core.contribution.publication.model.PublicationDetail;
-
+import org.silverpeas.core.importexport.versioning.Document;
+import org.silverpeas.core.node.importexport.NodePositionType;
+import org.silverpeas.core.pdc.pdc.model.ClassifyPosition;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlAttribute;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlElementWrapper;
+import javax.xml.bind.annotation.XmlRootElement;
+import java.util.List;
+
+@XmlRootElement(namespace = "http://www.silverpeas.org/exchange")
+@XmlAccessorType(XmlAccessType.NONE)
 public class PublicationType {
 
+  @XmlAttribute
   private int id = -1;
+  @XmlAttribute
   private String componentId;
+  @XmlElement(name = "publicationHeader")
   private PublicationDetail publicationDetail;
+  @XmlElement(name = "publicationContent")
   private PublicationContentType publicationContentType;
-  private AttachmentsType attachmentsType;
-  private NodePositionsType nodePositionsType;
-  private PdcPositionsType pdcPositionsType;
+  @XmlElementWrapper(name = "attachments")
+  @XmlElement(name = "attachment", namespace = "http://www.silverpeas.org/exchange")
+  private List<AttachmentDetail> attachmentsType;
+  @XmlElementWrapper(name = "documents")
+  @XmlElement(name = "document", namespace = "http://www.silverpeas.org/exchange")
+  private List<Document> documentsType;
+  @XmlElementWrapper(name = "topicPositions")
+  @XmlElement(name = "topicPosition", namespace = "http://www.silverpeas.org/exchange")
+  private List<NodePositionType> nodePositionsType;
+  @XmlElementWrapper(name = "pdcPositions")
+  @XmlElement(name = "pdcPosition", namespace = "http://www.silverpeas.org/exchange")
+  private List<ClassifyPosition> pdcPositionsType;
+  @XmlElement(name = "coordinatesPositions", namespace = "http://www.silverpeas.org/exchange")
   private CoordinatesPositionsType coordinatesPositionsType;
-  private DocumentsType documentsType;
 
-  /**
-   * @return
-   */
+  public PublicationType() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1186)